Src/FluentAssertions/Specialized/JsonValueExtensions.cs

```diff
@@ -0,0 +1,15 @@
+#if NET6_0_OR_GREATER
+
+using System.Text.Json.Nodes;
+
+namespace FluentAssertions.Specialized;
+
+internal static class JsonValueExtensions
+{
+    public static bool IsNumeric(this JsonValue value)
+    {
+        return value.TryGetValue<long>(out _) || value.TryGetValue<ulong>(out _) || value.TryGetValue<double>(out _);
```

Comment:
From https://stackoverflow.com/a/71590219
```suggestion
        return value.GetValue<JsonElement>().ValueKind == JsonValueKind.Number;
```

Reply:
How did you figure that out? 🫣

Reply:
Good ol' Stack Overflow.

Since `JsonValueKind.Number` is available in .NET 6 I thought there had to be a way to get it.